### ISSUE TITLE
Request new series

### ISSUE THREAD

**Issue by AWHIS-Manga** [AWHIS-Manga/Series#2]
Please leave the series ye want to be bulked here.

**Comment by kenthjunx** [comment]
One Punch-Man please...

**Comment by AWHIS-Manga** [comment]
o/
I presume its the oens being posted on teh OPM subreddit uploaded by /u/VibhavM ?

**Comment by kenthjunx** [comment]
yes, those that are flaired murata chapter. thank you very much!

**Comment by AWHIS-Manga** [comment]
@CalmRaptor seems like it already exists, links were on their discord and subreddit.

Murata chapters: https://cubari.moe/read/gist/OPM/  
Webcomic chapters: https://cubari.moe/read/gist/JYHJU/

**Comment by kenthjunx** [comment]
oh, i see. I tried looking for it before I requested, guess I didn't search enough. anyways, really, thank you very much for telling me.

**Comment by ghost** [comment]
demon slayer plz
